src/ctx.rs

```diff
@@ -75,6 +76,37 @@ pub fn small_memset<T: Clone + Copy, const UP_TO: usize, const WITH_DEFAULT: boo
     }
 }
 
+/// # Safety
+///
+/// `buf` must be correctly aligned and dereferenceable (but need not be
+/// initialized). `T` must not have a destructor (guaranteed by `T: Copy`).
+fn small_memset_raw<T: Clone + Copy, const UP_TO: usize, const WITH_DEFAULT: bool>(
```

Comment:
This needs to be unsafe, I think.

Reply:
Yeah, as is, but I don't think we need this function at all.

Reply:
I'm not using it right now, thought I would keep it anyway. Think we really should remove it? I don't want to rewrite it.

Reply:
Ah, I see.  I don't think it's good to keep around unsafe unused code, though.  Could you leave it in a branch so if we need it later it's easy to pull back in?

Reply:
Fair enough, left it in `sjc/txa_uninit`